docs/README.md

```diff
@@ -145,10 +145,11 @@ Feel free to engage with the community using our:
   - Archived and publicly searchable on [Linen](http://linen.venicedb.org)
 - [LinkedIn group](https://www.linkedin.com/groups/14129519/)
 - [GitHub issues](https://github.com/linkedin/venice/issues)
-- [Contributor's guide](dev_guide/how_to/contribution_agreement)
+- [Contributor's guide](https://venicedb.org/docs/dev_guide/how_to)
+- [Contributor's agreement](dev_guide/how_to/contribution_agreement)
```

Comment:
Can you please also fix this path? https://venicedb.org/docs/dev_guide/how_to/contribution_agreement

Reply:
Hmm... so the README page can resolve relative links when seen from venicedb.org but not when seen from the rendering on GitHub? Is that the issue?

Reply:
I verified that we can preserve links in README and the doc website by linking to the `.md` file.

Hey @eldernewborn. Can you please make these changes to your PR?
```
- [Contributor's guide](./dev_guide/how_to/how_to.md)
- [Contributor's agreement](./dev_guide/how_to/CONTRIBUTING.md)
```

I fixed all broken doc links that I will merge after your PR is done: https://github.com/linkedin/venice/pull/961/files